lib/backend/k8s/k8s.go

```diff
@@ -397,15 +415,19 @@ func (c *KubeClient) Delete(d *model.KVPair) error {
 	log.Debugf("Performing 'Delete' for %+v", d)
 	switch d.Key.(type) {
 	case model.GlobalConfigKey:
-		return c.globalConfigClient.Delete(d)
+		return c.globalFelixConfigClient.Delete(d)
```

Comment:
Hm, so this is nice because it makes it clear that it's for felix, however the model is called `GlobalConfig`, so it might be nice to keep this in-sync with what the model is called.

WDYT?